packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.tsx

```diff
@@ -286,31 +285,28 @@ export const Autocomplete = observer((properties: AutocompleteProps): JSX.Elemen
 							disableCollapse: true,
 							previewOnFocus: true,
 							valueProps,
-							theme: deepmerge(
-								{
-									components: {
-										facetGridOptions: {
-											columns: 3,
-											onClick: onClickFacetOption,
-										},
-										facetHierarchyOptions: {
-											hideCount: true,
-											onClick: onClickFacetOption,
-										},
-										facetListOptions: {
-											hideCheckbox: true,
-											hideCount: true,
-											onClick: onClickFacetOption,
-										},
-										facetPaletteOptions: {
-											hideLabel: true,
-											columns: 3,
-											onClick: onClickFacetOption,
-										},
+							theme: {
```

Comment:
Should these just live at the root of `components` in `theme` instead of being a nested `theme` of `facet`.